etcdserver/server.go

```diff
@@ -189,6 +189,11 @@ func NewServer(cfg *ServerConfig) (*EtcdServer, error) {
 		return nil, err
 	}
 
+	err = os.MkdirAll(cfg.MemberDir(), privateDirMode)
+	if err != nil && err != os.ErrExist {
+		return nil, err
+	}
```

Comment:
We used to use mkall to create member dir when creating member/wal. It might not be enough after we support separate wal dir.